### ISSUE TITLE
Fix incorrect instructions in Databases Lesson

### ISSUE THREAD

**Issue by KevinMulhern** [TheOdinProject/mini_courses#2]
Heres the report of the bug https://github.com/TheOdinProject/theodinproject/issues/844

**Comment by OmarMoataz** [comment]
So, this is the code that could produce the same result:
SELECT COUNT(Customers.customerID), Country
FROM Customers
JOIN Orders ON Customers.CustomerID = Orders.CustomerID	
GROUP BY Country
HAVING COUNT(*) > 10;

Note: there are no countries that have more than 60 orders as far as I know.
The easier way would be to remove that part.

**Comment by KevinMulhern** [comment]
If you're up for fixing it @OmarMoataz we would really appreciate it 😄

**Comment by OmarMoataz** [comment]
Yeah but I don't know if I should remove it or just add the join code?

**Comment by heyalvaro** [comment]
@KevinMulhern shouldn't this be modified in the [ruby on rails repo/database.md file](https://github.com/TheOdinProject/rails_course/blob/master/rails_programming/databases_and_activerecord/databases.md)?

**Comment by KevinMulhern** [comment]
Hey @heyalvaro, yeah I forgot the files in this course weren't live yet. We have a pull request to add them to the site. We are just waiting on some input from our designer for the badge that the course will have.

**Comment by KevinMulhern** [comment]
Hey @OmarMoataz, We're going back to one repo for the entire curriculum as we weren't getting the benefits we expected from the multiple repo structure for the curriculum.

I've already merged the databases course back into the curriculum repo. Don't worry your commits for this fix have been carried over though 😄 https://github.com/TheOdinProject/curriculum/pull/7854

I intend to get this new course live this weekend.

**Comment by OmarMoataz** [comment]
Thanks @KevinMulhern, glad I could help.
